lib/app/presentation/user/user_list/page/update_user.dart

```diff
@@ -0,0 +1,74 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_lerning_with_rest_api/app/core/values/app_colors.dart';
+import 'package:get/get.dart';
+
+import '../../../../core/data/model/user_list_model.dart';
+import '../../../../global_widgets/k_text_widget.dart';
+import '../ctrl/user_list_ctrl.dart';
+
+class UpdateUser extends StatefulWidget {
+  UpdateUser({Key? key, this.isEnable, this.data}) : super(key: key);
+
+  UserList? data;
+  bool? isEnable = false;
+
+  @override
+  State<UpdateUser> createState() => _UpdateUserState();
+}
+
+class _UpdateUserState extends State<UpdateUser> {
+  var ctrl = Get.put(UserListCtrl());
+
+  @override
+  void initState() {
+    ctrl.fullNameController.value.text = widget.data?.fullName?? "";
```

Comment:
Get the argument in the controller if you're using GetX.
And do the initialization stuff in onInit of the GetX controller.